### PR TITLE
feat: Mask SDK

### PR DESCRIPTION
**Pull request by Jack-Works**
Implemented: `Mask.persona` and `Mask.socialNetwork`

### CODE REVIEW

File: packages/mask-sdk/public-api/index.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
+ * @packageDocumentation
```
Comment (septs) [inline packages/mask-sdk/public-api/index.ts:2]:
```suggestion
 * @package Documentation
```

Comment (Jack-Works) [reply packages/mask-sdk/public-api/index.ts:2]: https://tsdoc.org/pages/tags/packagedocumentation/

File: packages/mask-sdk/public-api/index.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ * Mask SDK that provides the ability of interacting with Mask Network.
+ */
+
+declare namespace Mask {
+    /**
+     * Current Mask SDK version.
+     * @public
+     */
+    export const sdkVersion: number
```
Comment (septs) [inline packages/mask-sdk/public-api/index.ts:11]:
```suggestion
    export get sdkVersion(): number
```

File: packages/mask-sdk/public-api/mask-wallet.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        on(eventName: 'connect', listener: (message: ProviderConnectInfo) => void): this
+        on(eventName: 'disconnect', listener: (error: ProviderRpcError) => void): this
+        on(eventName: 'chainChanged', listener: (chainId: string) => void): this
+        on(eventName: 'accountsChanged', listener: (accounts: string[]) => void): this
+        /** @deprecated use disconnect event. */
+        on(eventName: 'close', listener: (...args: any[]) => void): this
+        /** @deprecated Use chainChanged event. */
+        on(eventName: 'networkChanged', listener: (...args: any[]) => void): this
+        /** @deprecated Use message event. */
+        on(eventName: 'notification', listener: (...args: any[]) => void): this
```
Comment (septs) [inline packages/mask-sdk/public-api/mask-wallet.ts:47]:
```suggestion
        /** @deprecated use `disconnect` event. */
        on(eventName: 'close', listener: (...args: any[]) => void): this
        /** @deprecated Use `chainChanged` event. */
        on(eventName: 'networkChanged', listener: (...args: any[]) => void): this
        /** @deprecated Use `message` event. */
        on(eventName: 'notification', listener: (...args: any[]) => void): this
```

**Pull request merged**